apps/server/src/scheduler.ts

```diff
@@ -0,0 +1,25 @@
+import schedule from 'node-schedule'
+
+import { fetchAndSaveSeoulData } from '@/controllers/v1/databatch'
+import { logger } from '@/utils/logger'
+
+export default class DateBatchScheduler {
+    private async updateSeoulData(): Promise<void> {
+        await fetchAndSaveSeoulData()
+    }
+
+    public run() {
+        schedule.scheduleJob('0 6 * * *', async () => {
+            try {
+                logger.info('Scheduler job started for Seoul cultural event data update')
+                await this.updateSeoulData()
```

Comment:
updateSeoulData 에서 특별한 작업이 없어서 바로 호출해도 될것 같아요
```suggestion
                await fetchAndSaveSeoulData()
```